aiokafka/consumer/fetcher.py

```diff
@@ -512,7 +512,10 @@ def on_done(fut, self=self):
                         # cancellation
                         if not task.done():
                             task.cancel()
-                        await task
+                        try:
+                            await task
```

Comment:
Let's imagine, that `_fetch_requests_routine` is cancelled while awating the task here. Then instead of terminating it will continue infinite loop causing routing to hang. Right?

Reply:
I did some research and it looks like there is no a single case in Python code base where they unconditionally swallow `CancelledError` exception. But it looks like a hard-to-hit race and we already suppress `CancelledError` in other places in `aiokafka`, so let's go this way for now and resolve it consistently later.